openstack/identity/v3/groups/results.go

```diff
@@ -75,6 +75,12 @@ type CreateResult struct {
 	groupResult
 }
 
+// DeleteResult is the response from a Delete operation. Call its ExtractErr
+// method to interpret it as a Group.
```

Comment:
Should be "Call its ExtractErr method to determine if the result succeeded or failed" or something along those lines.